db/migrations/1_create_posts.sql

```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS posts;
+
```

Comment:
unless you did something that keeps the previous posts, rather then dropping the table each time, you can just check if the table exists using IF EXISTS. so if the table does not exist THEN it creates a table ELSE it just gives a message back. could also add another table for users (people that created the post) => referring to the debug